src/PseudoSteppingSchemes.jl

```diff
@@ -55,7 +55,7 @@ function adaptive_step_parameters(pseudo_scheme, Xₙ, Gₙ, eki; Δt=1.0,
     return Xₙ₊₁, Δtₙ
 end
 
-function iglesias_2013_update(Xₙ, Gₙ, eki; Δtₙ=1.0, perturb_observation=false)
+function iglesias_2013_update(Xₙ, Gₙ, eki; Δtₙ=1.0, perturb_observation=true)
```

Comment:
If you implement types to represent these options then you can store data like `perturb_observation` inside the type!